generator/templates/client/client.gotmpl

```diff
@@ -32,7 +32,7 @@ type Client struct {
   formats strfmt.Registry
 }
 
-{{ range .Operations }}/*{{ if .Summary }}{{ .Summary }}{{ if .Description }}
+{{ range .Operations }}/* {{ if .Summary }}{{ pascalize .Name }}: {{ .Summary }}{{ if .Description }}
```

Comment:
I think it really wants to not have that space between /* and {{ 
it's probably golint or go vet that would be unhappy about the space